doc/guide/build_windows.hpp

```diff
@@ -150,10 +150,7 @@ If you prefer to use cmake GUI, follow these instructions:
     - If Boost is still not found, try adding the following variables and
       reconfigure:
       - Name: `Boost_INCLUDE_DIR`; type `PATH`; value `C:/boost/`
-      - Name: `Boost_PROGRAM_OPTIONS_LIBRARY_DEBUG`; type `FILEPATH`; value should be `C:/boost/lib64-msvc-14.2/boost_program_options-vc142-mt-gd-x64-1_71.lib`
-      - Name: `Boost_PROGRAM_OPTIONS_LIBRARY_RELEASE`; type `FILEPATH`; value should be `C:/boost/lib64-msvc-14.2/boost_program_options-vc142-mt-x64-1_71.lib`
       - Name: `Boost_SERIALIZATION_LIBRARY_DEBUG`; type `FILEPATH`; value should be `C:/boost/lib64-msvc-14.2/boost_serialization-vc142-mt-gd-x64-1_71.lib`
-      - Name: `Boost_SERIALIZATION_LIBRARY_RELEASE`; type `FILEPATH`; value should be `C:/boost/lib64-msvc-14.2/boost_program_options-vc142-mt-x64-1_71.lib`
```

Comment:
Maybe we should leave this line here for now?

Reply:
Of course